internal/pkg/cli/group/group.go

```diff
@@ -3,12 +3,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Package groups contains the names of command groups
+// Package groups contains the names of command groups.
 package group
 
+// Categories for each top level command in the CLI.
 const (
-	GettingStarted = "Getting Started ✨"
-	Develop        = "Develop 🔧"
+	GettingStarted = "Getting Started 🌱"
+	Develop        = "Develop ✨"
 	Settings       = "Settings ⚙️"
+	Operational    = "Operational 🧐"
```

Comment:
What do you think of “operations”??

Also what do these emojis look like on Linux??

Reply:
> What do you think of “operations”??

Done!

> Also what do these emojis look like on Linux??

Same UI as macOS 😎